proxy.js

```diff
@@ -0,0 +1,42 @@
+//ex. proxy server which I'd use in place of my index.js file if I were not using the cors npm package
+
+const express = require('express'),
+    request = require('request'),
+    bodyParser = require('body-parser'),
+    app = express();
+
+const myLimit = typeof(process.argv[2]) != 'undefined' ? process.argv[2] : '100kb';
+console.log('Using limit: ', myLimit);
+
+app.use(bodyParser.json({limit: myLimit}));
+
+app.use(express.static('public'));
+
+app.all('*', function (req, res, next) {
+
+    res.header("Access-Control-Allow-Origin", "*");
```

Comment:
be sure to add this to your service as well.

Reply:
yup it's there using default app.use(cors())